### PR TITLE
Lib work

### PR DESCRIPTION
**Pull request by Hellowlol**
- init pr.

### CODE REVIEW

File: plexapi/library.py [hunk 1]
```diff
@@ -204,6 +224,25 @@ def delete(self):
...
+           kwargs (dict): Dict of settings to edit.
+
+        """
+        part = '/library/sections/%s?%s' % (self.key, urlencode(kwargs))
+        self._server.query(part, method=self._server._session.put)
+
+        # Reload this way since the self.key dont have a full path, but is simply a id.
+        for s in self._server.library.sections():
+            if s.key == self.key:
+                return s
```
Comment (Hellowlol) [inline plexapi/library.py]: Update the cache top.

**Comment by Hellowlol** [comment]
Dont merge, still need to fix flake. I just send the PR so i could get some feedvack on the names and if add should be in server or library.

**Comment by Hellowlol** [comment]
@pkkid Can you review this when you have some spare time. Im looking for some feedback where its best to add the methods i have added.

**Comment by pkkid** [comment]
Hey, I'm still around.  I just burned myself out a bit working on this project and need a small break to relax and to catch up on other things.  I'll jump back onto plexapi stuff in another week or so. Sorry for the silence.

**Comment by Hellowlol** [comment]
@pkkid Take your time :)

**Comment by pkkid** [comment]
OMG those doc strings! :D

**Comment by Hellowlol** [comment]
Yeah...

**Pull request merged**